docs/appActions.md

```diff
@@ -1023,6 +1023,18 @@ Notifies autoplay has been blocked
 
 
 
+### savePassword() 
```

Comment:
This is actually only showing up because I fixed javadoc comment block on some of anthony's actions.  It's not introduced here.